### ISSUE TITLE
Is it possible to import results from _admin/openapi/convert into wiremock.org mappings ? In document I was able to find post __/admin/mappings/wiremock.org ( however this doesn’t exist now ) in the source code , please clarify , thanks !

### ISSUE THREAD

**Issue by Dalanit** [WireMock-Net/WireMock.Net#943]
Ask question related to the repo, e.g:
- How to achieve ... ?
- How does ... work?

Please be specific so we can provide the best answer possible.

**Comment by Dalanit** [comment]
Is there any other way to import wiremock.net mappings into wiremock.org mappings ?

**Comment by StefH** [comment]
No, there is no way to convert wiremock.net mappings to wiremock.org mappings.

(The other way is more or less possible : wiremock.net can understand and convert in a limited way to wiremock.org)

In the future, there will be a new spec which is compatible with both.
